prebid-server/developers/pbs-build-an-analytics-adapter.md

```diff
@@ -41,21 +43,183 @@ A simple [filesystem](https://github.com/prebid/prebid-server/tree/master/analyt
 
 It can be configured with:
 
-```
+```yaml
 analytics:
   file:
     filename: "path/to/file.log
 ```
+
 Prebid Server will then write sample log messages to the file you provided.
 
+### HTTP Analytics Module in PBS-Go
+The HTTP Analytics Module can send selected Analytics Events to a specified HTTP endpoint.
+
+The sample rate has to be between `0.0` (never sample) and `1.0` (always sample). The sample rate is always evaluated and defaults to `0.0`.
+
+The module uses [github.com/antonmedv/expr](github.com/antonmedv/expr) for complex filter options. The [analytics object](https://github.com/mllrsohn/prebid-server/tree/master/analytics/core.go) is always passed into the expression.
```

Comment:
@steffenmllr - we won't want this link to your personal fork. Looks to me like we just replace `mllrsohn` with 'prebid', yes?